assets/js/base/context/hooks/payment-methods/use-payment-method-interface.ts

```diff
@@ -171,10 +95,25 @@ export const usePaymentMethodInterface = () => {
 			displayPricesIncludingTax: getSetting(
 				'displayCartPricesIncludingTax',
 				false
-			),
+			) as boolean,
```

Comment:
not blocking for this PR, but wondering if we'd prefer to change `getSetting` to accept a type argument, so we could change this to 
```typescript
getSetting< boolean >(
				'displayCartPricesIncludingTax',
				false
			)
```
I don't know if it makes very much difference, but it seems to be a little cleaner. 🤷🏻

Reply:
https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/4250 issue created for this so no need to action here :)